ERCS/erc-7634.md

```diff
@@ -0,0 +1,208 @@
+---
+eip: 7634
+title: Limited Transferrable NFT
+description: An ERC-721 extension to limit transferability based on counts among NFTs
+author: Qin Wang (@qinwang-git), Saber Yu (@OniReimu), Shiping Chen <shiping.chen@data61.csiro.au>
+discussions-to: https://ethereum-magicians.org/t/erc-7634-limited-transferable-nft/18861
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-02-22
+requires: 165, 721
+---
+
+## Abstract
+
+This standard extends [ERC-721](./eip-721.md) to allow minters to customize the transferability of NFTs by setting parameters via TransferCount. The standard introduces an interface with internal functions to facilitate this functionality.
```

Comment:
This abstract certainly satisfies the "terse" requirement from EIP-1, but I don't think it gives enough detail (both technical and contextual) for the reader to really understand what you're proposing here.

For example, what's a TransferCount?